.github/workflows/ci.yml

```diff
@@ -5,8 +5,12 @@ on:
     branches:
       - main
 
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
```

Comment:
are we going to have another `yml` file for running experimental evals? don't see them being triggered anywhere here

Reply:
yeah wasnt sure if we wanted to be running these in CI. We might want to make this a separate yml and only have it run on merge into main